src/Applications/LDAS_App/GEOSldas_LDAS.rc

```diff
@@ -116,6 +116,18 @@ PERTURBATIONS: 0
 FIRST_ENS_ID: 0
 
 
+# ---- Ensemble forcing
+#
+#  NO  : Deterministic met forcing  (default)
+#  YES : Ensemble met forcing
+#        - Typically used in land-atmosphere data assimilation (LADAS) configuration. 
+#        - Must have forcing with matching ensemble ID for each land ensemble member.
+#        - MET_PATH and MET_TAG should not contain the ensemble IDs.
```

Comment:
the real forcing path should have 3 digits appends to MET_PATH

Reply:
@weiyuan-jiang, @saraqzhang: 

Thanks, @weiyuan-jiang, but I'm not sure I fully understand what you're trying to say.   Here's my guess:   

1. GEOSldas looks for the forcing files for ensemble member NNN in directory `MET_PATH[NNN]/`, where NNN is the 3-digit ensemble ID.  That is, the forcing files must be stored in member-specific directories.  
2. The user specifies `MET_PATH` in the exeinp file as before (i.e., without the 3 digits).

I'm guessing this is how the ensemble of forcing files is written by the ADAS?

If the above is correct, I'll add the info to GEOSldas_LDAS.rc.   Please let me know, thanks, R

Reply:
Yes, I think it is accurate